services/docker-compose-build.yml

```diff
@@ -137,8 +137,14 @@ services:
   static-webserver:
     image: local/static-webserver:${BUILD_TARGET:?build_target_required}
     build:
-      context: ../
-      dockerfile: services/static-webserver/Dockerfile
+      context: ./static-webserver/client
+      dockerfile: tools/qooxdoo-kit/builder/Dockerfile
+      args:
```

Comment:
basically, this PR performs qx compilation and keeps its out output inside the Dockerfile of the static-webserver

Reply:
yes, this allows docker buildx to build everything in parallel.